bdai_ros2_wrappers/bdai_ros2_wrappers/action.py

```diff
@@ -358,7 +358,8 @@ def __init__(self, action_type: Type, action_name: str, node: Optional[Node] = N
             kwargs: other keyword arguments are forwarded to the underlying action client.
             See `rclpy.action.ActionClient` documentation for further reference.
         """
-        node = node or scope.node()
+        if node is None:
```

Comment:
Is some of this a duplicate of https://github.com/bdaiinstitute/ros_utilities/pull/97 ?

Reply:
Indeed. I have to rebase now that #97 got in.